odoo/addons/base/models/ir_sequence.py

```diff
@@ -316,6 +316,11 @@ class IrSequenceDate_Range(models.Model):
     _rec_name = "sequence_id"
     _allow_sudo_commands = False
 
+    _unique_range_per_sequence = models.Constraint(
+        'UNIQUE(sequence_id, date_from, date_to)',
```

Comment:
Is `date_to` necessary here?

Reply:
`_create_date_range_seq` will always create the same sequence date range when there is a concurrency issue so `date_from` and `date_to` will always be the same.
We could thus only target the `date_from` yes but I feel that since it is not necessary for this bug to be fixed it we should not restrict it further.